### DISCUSSION TITLE
Extension for fix jasperreports custom fonts when generated uberjar

### DISCUSSION THREAD

**Discussion by kauanmocelin** [quarkiverse/quarkiverse#222] [General]
Hi, I was faced a very annoyng problem when using custom fonts with jasperreports and jasperreports-fonts libs, when quarkus generated uberjar the file jasperreports_extension.properties was override instead merged.
To solve this the only way I found, after a lot of research and try, was create a simple extension to merge this .properties file.

So, I know that already exists a extension for jasperreports(https://github.com/quarkiverse/quarkus-jasperreports), make sense create a new extension to specifically resolve this problem?

@gastaldi what do you think?

**Comment by gastaldi** [comment]
I think it would make sense to open an issue in the existing extension to fix this, which sounds like a bug
